core/templates/components/question-directives/modal-templates/remove-question-skill-link-modal.comonent.spec.ts

```diff
@@ -185,6 +185,10 @@ describe('Question deletion modal', () => {
 
       expect(componentInstance.questionRemovalIsAllowed).toBeTrue();
 
+      // The backend API request is mocked such that the skill is assigned to
```

Comment:
Can you explain where the mocking happens?

Reply:
The mocking is in the top part of the test suite i.e., in the `describe` section -- I have added a sentence to reflect that. Please take a look.